letsdrill.rb

```diff
@@ -1,24 +1,34 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+
+  if integer.between?(90,100)
```

Comment:
i haven't seen anyone use `.between` to solve this before i dont think. cool!

other options here might be `>=` or...

a range...

```ruby
case integer
when 90..100
  ...
end
```